### PR TITLE
Add HistogramPercentiles param for the aggregator

### PR DESCRIPTION
**Pull request by luciferous**
Example:

    <Plugin>
      ...
      <Module signalfx_metadata>
        Token "..."
        NotifyLevel "OKAY"
        HistogramPercentiles "0.95, 0.99"
      </Module>
    </Plugin>

### CODE REVIEW

File: src/collectd_dogstatsd.py [hunk 1]
```diff
@@ -64,6 +66,8 @@ def configure_callback(self, conf):
         for node in conf.children:
             if node.key == "DogStatsDPort":
                 self.listen_port = int(node.values[0])
+            if node.key == "HistogramPercentiles":
```
Comment (dgriff-tools) [inline src/collectd_dogstatsd.py]: change if to be elif

**Review by dgriff-tools — CHANGES_REQUESTED** [review]
looks good once you update that if block

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 ===================
 
 * Added python dependency six
+* add module key `HistogramPercentiles` for configuring histogram percentiles
```
Comment (mdubbyap) [inline CHANGELOG.md]: This should be new version not updating the existing.

**Comment by luciferous** [comment]
@dgriff-tools @mdubbyap Thanks for looking at this. I've made the requested changes.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,10 @@
-0.0.29 / 2016-11-02
+0.0.30 / TBD
```
Comment (mdubbyap) [inline CHANGELOG.md]: Just put in the current date please, thanks. Doesn't have to be the day it was merged.

**Comment by luciferous** [comment]
@dgriff-tools @mdubbyap Would you like me to squash the branch into one commit?

**Review by dgriff-tools — APPROVED** [review]

**Comment by dgriff-tools** [comment]
yes @luciferous . please squash your commits

**Comment by dgriff-tools** [comment]
had to revert this merge as you have some issues.
line 247 of collectd_dogstatsd.py should be self.config.histogram_percentiles
and you need flake8 to pass on line 90 in that same script

**Comment by luciferous** [comment]
Sorry about that @dgriff-tools - #51 addresses those.

**Pull request merged**